subsys/bluetooth/mesh/brg_cfg_srv.c

```diff
@@ -286,7 +287,7 @@ static int bridging_table_size_get(const struct bt_mesh_model *model, struct bt_
 }
 
 
-const struct bt_mesh_model_op _bt_mesh_brg_cfg_srv_op[] = {
+const struct bt_mesh_model_op bt_mesh_brg_cfg_srv_op[] = {
```

Comment:
I guess it is better to made vice versa changes and add `underscore` in header files. It will follow a common pattern. I suspect you took a look at sar model but seems sar server is the only one which uses it. The others follow the pattern.